openscenegraph is broken

Commit 0a77327e9ea3bd5624d2d016c86205a388fe65a6

```
[using autodetected 6 job(s)]
make[1]: Entering directory `/usr/lib/mxe'
uname -a
Linux sd-68075 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u1 x86_64 GNU/Linux
git show-branch --list --reflog=1
  [refs/heads/master@{0}] (6 hours ago) pull: Merge made by the 'recursive' strategy.
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID:	Debian
Description:	Debian GNU/Linux 7.9 (wheezy)
Release:	7.9
Codename:	wheezy
autoconf --version 2>/dev/null | head -1
autoconf (GNU Autoconf) 2.69
automake --version 2>/dev/null | head -1
automake (GNU automake) 1.11.6
rm -rf   '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static'
mkdir -p '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static'
cd '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static' &&         unzip -q '/usr/lib/mxe/pkg/OpenSceneGraph-3.4.0.zip'
cd '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0'
mkdir '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
cd '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build' && cmake '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0' -DCMAKE_TOOLCHAIN_FILE='/usr/lib/mxe/usr/i686-w64-mingw32.static/share/cmake/mxe-conf.cmake' -DCMAKE_CXX_FLAGS='-D__STDC_CONSTANT_MACROS' -DCMAKE_HAVE_PTHREAD_H=OFF -DPKG_CONFIG_EXECUTABLE='/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-pkg-config' -DDYNAMIC_OPENTHREADS=OFF -DDYNAMIC_OPENSCENEGRAPH=OFF -DBUILD_OSG_APPLICATIONS=OFF -DPOPPLER_HAS_CAIRO_EXITCODE=0 -D_OPENTHREADS_ATOMIC_USE_GCC_BUILTINS_EXITCODE=1 -D_OPENTHREADS_ATOMIC_USE_WIN32_INTERLOCKED=1
-- The C compiler identification is GNU 5.2.0
-- The CXX compiler identification is GNU 5.2.0
-- Check for working C compiler: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-gcc
-- Check for working C compiler: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++
-- Check for working CXX compiler: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Found Threads: TRUE  
-- Found OpenGL: opengl32  
-- Performing Test GL_HEADER_HAS_GLINT64
-- Performing Test GL_HEADER_HAS_GLINT64 - Failed
-- Performing Test GL_HEADER_HAS_GLUINT64
-- Performing Test GL_HEADER_HAS_GLUINT64 - Failed
-- 32 bit architecture detected
-- Could NOT find Freetype (missing:  FREETYPE_INCLUDE_DIRS) (found version "2.6.1")
-- Found JPEG: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libjpeg.a  
-- Found Jasper: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libjasper.a (found version "1.900.1") 
-- Found LibXml2: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libxml2.a (found version "2.9.2") 
-- Found ZLIB: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libz.a (found version "1.2.8") 
-- Found GDAL: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libgdal.a  
-- Found PkgConfig: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-pkg-config (found version "0.28") 
-- checking for module 'gta'
--   found gta, version 1.0.7
-- Found CURL: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libcurl.a (found version "7.45.0") 
-- Found GStreamer: /usr/lib/mxe/usr/i686-w64-mingw32.static/include/gstreamer-1.0 (found version "1.4.5") 
-- Found GLIB: /usr/lib/mxe/usr/i686-w64-mingw32.static/include/glib-2.0;/usr/lib/mxe/usr/i686-w64-mingw32.static/lib/glib-2.0/include (found version "2.44.1") 
-- Found SDL2: mingw32;/usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libSDL2main.a;/usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libSDL2.a  
== Custom MXE File: /usr/lib/mxe/src/cmake/modules/FindSDL.cmake
-- checking for module 'sdl'
--   found sdl, version 1.2.15
-- checking for module 'cairo'
--   found cairo, version 1.14.2
-- checking for module 'poppler-glib'
--   found poppler-glib, version 0.30.0
-- Performing Test POPPLER_HAS_CAIRO
-- Performing Test POPPLER_HAS_CAIRO - Success
-- checking for module 'librsvg-2.0>=2.35'
--   found librsvg-2.0, version 2.40.5
-- checking for module 'gtk+-2.0'
--   found gtk+-2.0, version 2.24.22
-- checking for module 'gtkglext-win32-1.0'
--   found gtkglext-win32-1.0, version 1.2.0
-- Found PNG: /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libpng.a (found version "1.6.18") 
== Custom MXE File: /usr/lib/mxe/src/cmake/modules/FindTIFF.cmake
-- checking for module 'libtiff-4'
--   found libtiff-4, version 4.0.6
-- Performing Test _OPENTHREADS_ATOMIC_USE_GCC_BUILTINS
-- Performing Test _OPENTHREADS_ATOMIC_USE_GCC_BUILTINS - Failed
-- Performing Test _OPENTHREADS_ATOMIC_USE_MIPOSPRO_BUILTINS
-- Performing Test _OPENTHREADS_ATOMIC_USE_MIPOSPRO_BUILTINS - Failed
-- Performing Test _OPENTHREADS_ATOMIC_USE_SUN
-- Performing Test _OPENTHREADS_ATOMIC_USE_SUN - Failed
-- Performing Test _OPENTHREADS_ATOMIC_USE_BSD_ATOMIC
-- Performing Test _OPENTHREADS_ATOMIC_USE_BSD_ATOMIC - Failed
-- Configuring done
-- Generating done
-- Build files have been written to: /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build
make -C '/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build' -j '6' install VERBOSE=1
make[2]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
/usr/bin/cmake -H/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0 -B/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[3]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make -f src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/build.make src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/depend
make[4]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0 /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/win32 /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/DependInfo.cmake --color=
Dependee "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/DependInfo.cmake" is newer than depender "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/depend.internal".
Dependee "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/depend.internal".
Scanning dependencies of target OpenThreads
make[4]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make -f src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/build.make src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/build
make[4]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] [  0%] [  0%] [  0%] [  0%] Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/Win32ThreadBarrier.obj
Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/Win32Thread.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/Win32ThreadBarrier.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/win32/Win32ThreadBarrier.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/Win32Thread.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/win32/Win32Thread.cpp
Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/Win32Mutex.obj
Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/__/common/Version.obj
Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/__/common/Atomic.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/Win32Mutex.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/win32/Win32Mutex.cpp
Building CXX object src/OpenThreads/win32/CMakeFiles/OpenThreads.dir/WIN32Condition.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/__/common/Version.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/common/Version.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/__/common/Atomic.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/common/Atomic.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOPENTHREADS_EXPORTS -DWIN32_LEAN_AND_MEAN -D_WIN32_WINNT=0x0400 -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/OpenThreads.dir/includes_CXX.rsp   -o CMakeFiles/OpenThreads.dir/WIN32Condition.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/win32/WIN32Condition.cpp
/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/common/Atomic.cpp:19:0: warning: ignoring #pragma intrinsic  [-Wunknown-pragmas]
 #pragma intrinsic(_InterlockedAnd)
 ^
/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/common/Atomic.cpp:20:0: warning: ignoring #pragma intrinsic  [-Wunknown-pragmas]
 #pragma intrinsic(_InterlockedOr)
 ^
/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/OpenThreads/common/Atomic.cpp:21:0: warning: ignoring #pragma intrinsic  [-Wunknown-pragmas]
 #pragma intrinsic(_InterlockedXor)
 ^
Linking CXX static library ../../../lib/libOpenThreads.a
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/bin/cmake -P CMakeFiles/OpenThreads.dir/cmake_clean_target.cmake
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/OpenThreads/win32 && /usr/bin/cmake -E cmake_link_script CMakeFiles/OpenThreads.dir/link.txt --verbose=1
/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ar cr ../../../lib/libOpenThreads.a  CMakeFiles/OpenThreads.dir/WIN32Condition.obj CMakeFiles/OpenThreads.dir/Win32Mutex.obj CMakeFiles/OpenThreads.dir/Win32Thread.obj CMakeFiles/OpenThreads.dir/Win32ThreadBarrier.obj CMakeFiles/OpenThreads.dir/__/common/Version.obj CMakeFiles/OpenThreads.dir/__/common/Atomic.obj
/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ranlib ../../../lib/libOpenThreads.a
make[4]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] Built target OpenThreads
make -f src/osg/CMakeFiles/osg.dir/build.make src/osg/CMakeFiles/osg.dir/depend
make[4]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0 /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg/CMakeFiles/osg.dir/DependInfo.cmake --color=
Dependee "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg/CMakeFiles/osg.dir/DependInfo.cmake" is newer than depender "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg/CMakeFiles/osg.dir/depend.internal".
Dependee "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg/CMakeFiles/osg.dir/depend.internal".
Scanning dependencies of target osg
make[4]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make -f src/osg/CMakeFiles/osg.dir/build.make src/osg/CMakeFiles/osg.dir/build
make[4]: Entering directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] [  0%] /usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] [  0%] /usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
Building CXX object src/osg/CMakeFiles/osg.dir/ArgumentParser.obj
Building CXX object src/osg/CMakeFiles/osg.dir/ApplicationUsage.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ArgumentParser.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ArgumentParser.cpp
Building CXX object src/osg/CMakeFiles/osg.dir/Array.obj
[  0%] cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/AlphaFunc.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/AlphaFunc.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ApplicationUsage.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ApplicationUsage.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Array.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Array.cpp
Building CXX object src/osg/CMakeFiles/osg.dir/AnimationPath.obj
Building CXX object src/osg/CMakeFiles/osg.dir/ArrayDispatchers.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ArrayDispatchers.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ArrayDispatchers.cpp
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/AnimationPath.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/AnimationPath.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] Building CXX object src/osg/CMakeFiles/osg.dir/AudioStream.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/AudioStream.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/AudioStream.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  0%] Building CXX object src/osg/CMakeFiles/osg.dir/AutoTransform.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/AutoTransform.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/AutoTransform.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 1
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/Billboard.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Billboard.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Billboard.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BlendColor.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BlendColor.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BlendColor.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BlendEquation.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BlendEquation.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BlendEquation.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BlendEquationi.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BlendEquationi.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BlendEquationi.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BlendFunc.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BlendFunc.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BlendFunc.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BlendFunci.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BlendFunci.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BlendFunci.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BufferIndexBinding.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BufferIndexBinding.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BufferIndexBinding.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/BufferObject.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/BufferObject.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/BufferObject.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/Callback.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Callback.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Callback.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/Capability.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Capability.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Capability.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/Camera.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Camera.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Camera.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/CameraView.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CameraView.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CameraView.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/ClampColor.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClampColor.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClampColor.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/ClearNode.obj
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClearNode.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClearNode.cpp
[  1%] Building CXX object src/osg/CMakeFiles/osg.dir/ClipControl.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClipControl.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClipControl.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 2
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ClipNode.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClipNode.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClipNode.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ClipPlane.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClipPlane.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClipPlane.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ClusterCullingCallback.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ClusterCullingCallback.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ClusterCullingCallback.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] [  2%] Building CXX object src/osg/CMakeFiles/osg.dir/CollectOccludersVisitor.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CollectOccludersVisitor.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CollectOccludersVisitor.cpp
Building CXX object src/osg/CMakeFiles/osg.dir/ColorMask.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ColorMask.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ColorMask.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ColorMaski.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ColorMaski.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ColorMaski.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ColorMatrix.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ColorMatrix.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ColorMatrix.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ComputeBoundsVisitor.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ComputeBoundsVisitor.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ComputeBoundsVisitor.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ConvexPlanarOccluder.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ConvexPlanarOccluder.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ConvexPlanarOccluder.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/ConvexPlanarPolygon.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/ConvexPlanarPolygon.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/ConvexPlanarPolygon.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/CoordinateSystemNode.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CoordinateSystemNode.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CoordinateSystemNode.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/CopyOp.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CopyOp.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CopyOp.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/CullFace.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CullFace.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CullFace.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  2%] Building CXX object src/osg/CMakeFiles/osg.dir/CullingSet.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CullingSet.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CullingSet.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 3
[  3%] Building CXX object src/osg/CMakeFiles/osg.dir/CullSettings.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CullSettings.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CullSettings.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  3%] Building CXX object src/osg/CMakeFiles/osg.dir/CullStack.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/CullStack.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/CullStack.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  3%] Building CXX object src/osg/CMakeFiles/osg.dir/DeleteHandler.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/DeleteHandler.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/DeleteHandler.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  3%] Building CXX object src/osg/CMakeFiles/osg.dir/Depth.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/Depth.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/Depth.cpp
/usr/bin/cmake -E cmake_progress_report /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/CMakeFiles 
[  3%] Building CXX object src/osg/CMakeFiles/osg.dir/DisplaySettings.obj
cd /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build/src/osg && /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-g++   -DOSG_LIBRARY_STATIC -D__STDC_CONSTANT_MACROS -Wall -Wparentheses -Wno-long-long -Wno-import -pedantic -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wunused -O3 -DNDEBUG @CMakeFiles/osg.dir/includes_CXX.rsp   -o CMakeFiles/osg.dir/DisplaySettings.obj -c /usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/DisplaySettings.cpp
/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0/src/osg/DisplaySettings.cpp:27:20: fatal error: Windows.h: No such file or directory
compilation terminated.
make[4]: *** [src/osg/CMakeFiles/osg.dir/DisplaySettings.obj] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make[3]: *** [src/osg/CMakeFiles/osg.dir/all] Error 2
make[3]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/usr/lib/mxe/tmp-openscenegraph-i686-w64-mingw32.static/OpenSceneGraph-3.4.0.build'
make[1]: *** [build-only-openscenegraph_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory `/usr/lib/mxe'

real	0m25.062s
user	1m20.565s
sys	0m5.976s
```

--- a/src/openscenegraph-1-fix-case-in-headers.patch
+++ b/src/openscenegraph-1-fix-case-in-headers.patch
@@ -1,0 +1,58 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 31fe54c6c787bc3b66dd22a17528aeaeb0544752 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Fri, 9 Oct 2015 11:13:12 +0200
+Subject: [PATCH] fix case in headers windows.h, winsock2.h
+
+---
+ examples/osghangglide/hat.cpp |    2 +-
+ src/osg/DisplaySettings.cpp   |    2 +-
+ src/osgPlugins/ply/typedefs.h |    4 ++--
+ 3 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/examples/osghangglide/hat.cpp b/examples/osghangglide/hat.cpp
+index 4ef9e79..2884770 100644
+--- a/examples/osghangglide/hat.cpp
++++ b/examples/osghangglide/hat.cpp
+@@ -17,7 +17,7 @@
+ */
+ 
+ #ifdef _MSC_VER
+-#include <Windows.h>
++#include <windows.h>
+ #pragma warning( disable : 4244 )
+ #endif
+ 
+diff --git a/src/osg/DisplaySettings.cpp b/src/osg/DisplaySettings.cpp
+index 8ebae0d..8731c10 100644
+--- a/src/osg/DisplaySettings.cpp
++++ b/src/osg/DisplaySettings.cpp
+@@ -24,7 +24,7 @@ using namespace osg;
+ using namespace std;
+ 
+ #if defined(WIN32) && !defined(__CYGWIN__)
+-#include<Windows.h>
++#include<windows.h>
+ extern "C" { OSG_EXPORT DWORD NvOptimusEnablement=0x00000001; }
+ #else
+ extern "C" { int NvOptimusEnablement=0x00000001; }
+diff --git a/src/osgPlugins/ply/typedefs.h b/src/osgPlugins/ply/typedefs.h
+index 4d6c833..7a1e334 100644
+--- a/src/osgPlugins/ply/typedefs.h
++++ b/src/osgPlugins/ply/typedefs.h
+@@ -13,8 +13,8 @@
+ #define MESH_TYPEDEFS_H
+ 
+ #   if defined(_MSC_VER)
+-#      include <Winsock2.h>
+-#      include <Windows.h>
++#      include <winsock2.h>
++#      include <windows.h>
+ #   endif
+ 
+ #   include <osg/Notify>
+-- 
+1.7.10.4
+

--- a/src/openscenegraph-2-fix-case-in-library-ws2-32.patch
+++ b/src/openscenegraph-2-fix-case-in-library-ws2-32.patch
@@ -1,0 +1,56 @@
+This file is part of MXE.
+See index.html for further information.
+
+From c78dde633089cdff3142eb30cc01c5bd77a3c617 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Fri, 9 Oct 2015 12:54:07 +0200
+Subject: [PATCH] fix case in library ws2_32
+
+---
+ src/osgPlugins/ZeroConfDevice/CMakeLists.txt |    2 +-
+ src/osgPlugins/osc/CMakeLists.txt            |    2 +-
+ src/osgPlugins/vrml/CMakeLists.txt           |    2 +-
+ 3 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/src/osgPlugins/ZeroConfDevice/CMakeLists.txt b/src/osgPlugins/ZeroConfDevice/CMakeLists.txt
+index 5287cfc..fab449e 100644
+--- a/src/osgPlugins/ZeroConfDevice/CMakeLists.txt
++++ b/src/osgPlugins/ZeroConfDevice/CMakeLists.txt
+@@ -19,7 +19,7 @@ IF(WIN32)
+         mdns_win/dns_sd.h
+         mdns_win/dns-sd.c
+     )
+-    SET(TARGET_EXTERNAL_LIBRARIES "${TARGET_EXTERNAL_LIBRARIES};Ws2_32.lib;winmm")
++    SET(TARGET_EXTERNAL_LIBRARIES "${TARGET_EXTERNAL_LIBRARIES};ws2_32.lib;winmm")
+     SET(TARGET_LIBRARIES_VARS ZEROCONF_LIBRARY)
+     ADD_DEFINITIONS(-DNOT_HAVE_GETOPT)
+     ADD_DEFINITIONS(-DNOT_HAVE_SETLINEBUF)
+diff --git a/src/osgPlugins/osc/CMakeLists.txt b/src/osgPlugins/osc/CMakeLists.txt
+index 6bf6127..0529e17 100644
+--- a/src/osgPlugins/osc/CMakeLists.txt
++++ b/src/osgPlugins/osc/CMakeLists.txt
+@@ -35,7 +35,7 @@ if(WIN32 AND NOT ANDROID)
+         ip/win32/NetworkingUtils.cpp
+         ip/win32/UdpSocket.cpp
+     )
+-    SET(TARGET_EXTERNAL_LIBRARIES "${TARGET_EXTERNAL_LIBRARIES};Ws2_32.lib;winmm")
++    SET(TARGET_EXTERNAL_LIBRARIES "${TARGET_EXTERNAL_LIBRARIES};ws2_32.lib;winmm")
+ ELSE()
+     SET(TARGET_SRC
+         ${TARGET_SRC}
+diff --git a/src/osgPlugins/vrml/CMakeLists.txt b/src/osgPlugins/vrml/CMakeLists.txt
+index 867e0df..5e5eb26 100644
+--- a/src/osgPlugins/vrml/CMakeLists.txt
++++ b/src/osgPlugins/vrml/CMakeLists.txt
+@@ -27,7 +27,7 @@ IF (WIN32)
+         PNG_LIBRARY
+         ZLIB_LIBRARY)
+     SET(TARGET_EXTERNAL_LIBRARIES
+-        Ws2_32.lib)
++        ws2_32.lib)
+ ELSE()
+     SET(TARGET_LIBRARIES_VARS
+         OPENVRML_LIBRARY)
+-- 
+1.7.10.4
+

--- a/src/openscenegraph-3-fix-pointer-to-int-conversion-on-64bit.patch
+++ b/src/openscenegraph-3-fix-pointer-to-int-conversion-on-64bit.patch
@@ -1,0 +1,28 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 9858a76d8daebe3e26f67eb530c89a8f00d5cd28 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Fri, 9 Oct 2015 14:00:54 +0200
+Subject: [PATCH] fix pointer to int conversion on 64bit
+
+---
+ src/osgPlugins/osgjs/WriteVisitor.cpp |    2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/osgPlugins/osgjs/WriteVisitor.cpp b/src/osgPlugins/osgjs/WriteVisitor.cpp
+index 9f2b3c7..98e0c19 100644
+--- a/src/osgPlugins/osgjs/WriteVisitor.cpp
++++ b/src/osgPlugins/osgjs/WriteVisitor.cpp
+@@ -225,7 +225,7 @@ JSONObject* createImage(osg::Image* image, bool inlineImages, int maxTextureDime
+             // no image file so use this inline name image and create a file
+             std::stringstream ss;
+             ss << osgDB::getFilePath(baseName) << osgDB::getNativePathSeparator();
+-            ss << (long int)image << ".inline_conv_generated.png"; // write the pointer location
++            ss << (long long int)image << ".inline_conv_generated.png"; // write the pointer location
+             std::string filename = ss.str();
+             if (osgDB::writeImageFile(*image, filename)) {
+                 image->setFileName(filename);
+-- 
+1.7.10.4
+

